teams/blau/beschleunigung.m

```diff
@@ -79,13 +76,12 @@ function setupNodeGrid()
                 nodeGrid(x,y).hCost = 0;
                 nodeGrid(x,y).fCost = 0;
                 nodeGrid(x,y).gCost = 0;
+                nodeGrid(x,y).mineCost = 0;
                 
                 %Je dichter an Mine, desto teurer!
-                for i=1:spiel.n_mine
-                   if norm([x,y]-spiel.mine(i).pos) < 1
-                      nodeGrid(x,y).fCost = 1/(spiel.mine_radius+norm([x,y]-spiel.mine(i).pos));
-                   end
-                end
+                %for i=1:spiel.n_mine
+                %   nodeGrid(x,y).mineCost = nodeGrid(x,y).mineCost + pow2(1/norm(nodeGrid(x,y).worldPos-spiel.mine(i).pos));
+                %end
                 
```

Comment:
momentan Auskommentiert da Endlosschleife
der Abstand jeder mine invertiert und ins quadraht wird in mineCost aufaddiert